.github/workflows/release_ci.yml

```diff
@@ -20,5 +20,4 @@ jobs:
     with:
       cloud: azure
       max_parallel: 10
-      tf_version: 1.2 1.3 1.4
```

Comment:
Why you remove the tf_version here?

Reply:
Following the discussion yesterday on planning - we will run `apply` only on the latest TF version, if you do not specify any here it will automatically pick up `latest`, if you put here any version, `1.5` in example, you will have to update the code each time something new is released

Reply:
Got it!